src/javascript/app/pages/trade/price.js

```diff
@@ -151,6 +156,21 @@ const Price = (() => {
             return;
         }
 
+        // hide all containers except current one
+        if (position !== 'middle') {
+            const ele = document.getElementById('price_container_middle');
+            if ($(ele).is(':visible')) {
+                $(ele).fadeOut(200);
+            }
+        } else {
+            const ele_top = document.getElementById('price_container_top');
+            const ele_bottom = document.getElementById('price_container_bottom');
+            if ($(ele_top).is(':visible') || $(ele_bottom).is(':visible')) {
+                $(ele_top).fadeOut(200);
+                $(ele_bottom).fadeOut(200);
+            }
```

Comment:
if all of these elements are going to be jquery why get them from javascript at all? can just be `const $ele_top = $('#price_container_top');` etc